Book list creation

eng-mohamed-eldeeb opened the pull request:
Create a form that takes the book name and the author name. When the form is submitted the book appears with a remove button and the book is added to local storage. when the remove button is clicked the book is removed from the list and the local storage.

eng-mohamed-eldeeb committed:
Inastall linters

eng-mohamed-eldeeb committed:
Finsh the project

eng-mohamed-eldeeb committed:
Finsh html and css

eng-mohamed-eldeeb committed:
Fix a JS typing error

eng-mohamed-eldeeb committed:
Add more style and Fix linter error

Sirvelasque committed:
storage the books localy

added a localStorage use to the books list and made it update by clicking on the remove and add events

Sirvelasque committed:
change the way to remove books

move the remove books function into a separate function to later be called by the remove button

eng-mohamed-eldeeb committed:
Add to the README file

eng-mohamed-eldeeb committed:
Fix linter error

mikethreels commented on main.js:
```diff
@@ -0,0 +1,62 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+// create the books, add them the book list and local storage
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const newBooke = new Book(title.value, author.value);
+  books.push(newBooke);
+  creatAndAddBooks(newBooke, bookList);
+  localStorage.books = JSON.stringify(books);
+});
```
- [ ] you are adding the book to the local storage correctly which is great 👏 however when reloading the page all the books that were saved disappear anyway, this is due to the fact that you are not getting your data from the local storage when reloading the page. please make sure to fix this before requesting for another code review.

mikethreels reviewed (changes requested):
Hi Team, 

Good job creating the functions for adding and deleting the books 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
get books list from local storage

Sirvelasque committed:
create books from the local storage

Sirvelasque committed:
Update main.js

Sirvelasque committed:
Update main.js

er-contreras reviewed (changes requested):
Hi @eng-mohamed-eldeeb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

![almost there](https://media.giphy.com/media/7JsChhLbpj0rJtzimC/giphy.gif)

## Required Changes ♻️

- [ ] There is a typo in your HTML file you wrote ' Insdex.html ' instead of  ' index.html ' that causes that live-server don't load it correctly.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
change file name so it can be deployed

n/a

Sirvelasque committed:
prev

Sirvelasque committed:
change done

Sirvelasque committed:
condition to take values from storage

the array is just created but it only takes values from the local storage if they exist

Sirvelasque committed:
Update main.js

TedLivist reviewed (approved):
## STATUS: APPROVED ✔️ ✔️

Hi @eng-mohamed-eldeeb and @Sirvelasque,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
